<?xml version="1.0"?>

<launch>
    <arg name="machine"/>
    <rosparam command="load" file="$(find lowlevel)/params/robot_config.yaml" ns="$(env ROBOT)"/>
    <node pkg="lowlevel" type="robot_controller" name="robot_controller" machine="$(arg machine)" respawn="true" output="screen">
        <param name="robot_name" value="$(env ROBOT)"/>
    </node>
</launch>
